src/fparser/two/Fortran2003.py

```diff
@@ -1318,7 +1318,6 @@ def match(reader):
                                 Component_Part, Type_Bound_Procedure_Part],
                                End_Type_Stmt, reader,
                                match_names=True,
-                               set_unspecified_end_name=True  # C431
```

Comment:
Note: C431 states:

> (R425) If EXTENDS appears, SEQUENCE shall not appear.

To be honest, I couldn't see how that constraint applies to the ``set_unspecified_end_name``.

Reply:
I've now changed this (and removed the additional ",")